ruby/pom.xml

```diff
@@ -78,7 +78,7 @@
         <dependency>
             <groupId>com.google.protobuf</groupId>
             <artifactId>protobuf-java</artifactId>
-            <version>3.0.0-pre</version>
+            <version>3.0.0-alpha-3-pre</version>
```

Comment:
is there a way to automate this?  the last time I used java it was early ant days, so I don't know maven at all